packages/design-system/src/themes/default/index.css

```diff
@@ -2,10 +2,29 @@
 @import "variables.css";
 
 :root {
+
+  /* new variables */
```

Comment:
Let's use a timeless comment - "new variables" doesn't work two months from now, and means very little - how about "theme variables"?